src/lib/markbind/src/preprocessors/componentPreprocessor.js

```diff
@@ -123,39 +120,42 @@ function _getSrcFlagsAndFilePaths(element, context, config) {
  * and adds the appropriate include.
  */
 function _preProcessPanel(node, context, config, parser) {
-  const element = node;
-
-  const hasSrc = _.has(element.attribs, 'src');
+  const hasSrc = _.has(node.attribs, 'src');
   if (!hasSrc) {
-    if (element.children && element.children.length > 0) {
+    if (node.children && node.children.length > 0) {
       // eslint-disable-next-line no-use-before-define
-      element.children = element.children.map(e => preProcessComponent(e, context, config, parser));
+      node.children = node.children.map(e => preProcessComponent(e, context, config, parser));
     }
 
-    return element;
+    return node;
   }
 
   const {
     isUrl,
     hash,
     filePath,
     actualFilePath,
-  } = _getSrcFlagsAndFilePaths(element, context, config);
+  } = _getSrcFlagsAndFilePaths(node, context, config);
 
-  const fileExistsNode = _getFileExistsNode(element, context, config, parser, actualFilePath);
+  const fileExistsNode = _getFileExistsNode(node, context, config, parser, actualFilePath);
   if (fileExistsNode) {
     return fileExistsNode;
   }
 
   if (!isUrl && hash) {
-    element.attribs.fragment = hash.substring(1);
+    node.attribs.fragment = hash.substring(1);
   }
 
-  element.attribs.src = filePath;
+  const { fragment } = node.attribs;
+  const resultPath = utils.setExtension(path.relative(config.rootPath, filePath), '._include_.html');
```

Comment:
Would it make sense to rename `resultPath` -> `relativePath` and `baseUrlPrependedPath` -> `absolutePath`? I feel like that might better reflect what they are.